plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -119,6 +143,7 @@ func (k *KinesisConsumer) connect(ac telegraf.Accumulator) error {
 		consumer.WithClient(client),
 		consumer.WithShardIteratorType(k.ShardIteratorType),
 		consumer.WithStore(k),
+		consumer.WithLogger(&TelegrafLoggerWrapper{k.Log}),
```

Comment:
Could we pass `cfg.Logger` here since it is already wrapped, or would that cause issues?